ganttproject/src/main/java/biz/ganttproject/customproperty/CustomColumnsValues.java

```diff
@@ -160,4 +157,17 @@ static String getValueAsString(Object value) {
     }
     return result;
   }
+
+  @Override
+  public boolean equals(Object object) {
+    if (this == object) return true;
+    if (!(object instanceof CustomColumnsValues)) return false;
+    CustomColumnsValues columnsValues = (CustomColumnsValues) object;
+    return mapCustomColumnValue.equals(columnsValues.mapCustomColumnValue) && myManager.equals(columnsValues.myManager);
```

Comment:
Seems like the manager is never set to null, is that correct? Or should I add and extra check for that?

Reply:
I don't think that it is needed at all. Just remove it from equals.